data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReportingAppContext.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for feedback reporting app info represented in the backend domain model
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeFeedbackReportingAppContext(
+
+  @Json(name = "entry_point") val entryPoint: String?,
+  // Maps a string representing the topic progress to the number of topics with that status.
```

Comment:
Actually, this confuses me. Shouldn't topic progress just be a list of completed explorations? I think that's all we need actually.

Reply:
My hesitation is that this might get really long as more topics are added, but I'd agree that this would be more useful.
Updated

Reply:
It shouldn't be too long (e.g. <1000). If that scales poorly, we can address it later.